### ISSUE TITLE
Fix double ellipsis for selectize

### ISSUE THREAD

**Issue by amgalanbadmaev** [freeats/freeats#250]
**How it works**
If you select option with long name, first ellipsis adds, after selection second option - second ellipsis adds and ignores our right-padding for chevron:

![CleanShot 22 11 2024 at 11-01](https://github.com/user-attachments/assets/5f937b19-d6c7-4363-8dd5-13f5a4fc9e3f)

**How it should work**
Second ellipsis shouldn't appear and text shouldn't ignore select's right-padding.

**Comment by wrmk** [comment]
yeah, we [fixed ](https://github.com/toughbyte/hub/pull/15709/files)this in the internal project at the end of august and did not moved into ats.
will do
